source/Camera/Basic/CaptureTutorial.md

```diff
@@ -0,0 +1,214 @@
+## Introduction
+
+This tutorial describes how to use Zivid SDK to capture point clouds and 2D images.
+
+1. [Initialize](#initialize)
+2. [Connect](#connect)
+   1. [Specific Camera](#connect---specific-camera)
+   2. [Virtual Camera](#connect---virtual-camera)
+3. [Configure](#configure)
```

Comment:
I vote for this structure:

3. [Configure](#configure)
   1. [Capture Assistant](#capture-assistant)
   2. [Manual Configuration](#manual-configuration)
      1. [Single](#single-frame)
      2. [HDR](#hdr-frame)
      3. [2D](#2d-settings)
   3. [From File](#from-file)